### PR TITLE
QML/QC/QChem icons in demos page

### PR DESCRIPTION
**Pull request by ikurecic**
Inserting existing illustrations for QML/QC/QChem links in the Demos page.

### CODE REVIEW

File: demonstrations.rst [hunk 1]
```diff
@@ -119,7 +119,7 @@ Demos
                             <div class="text-center d-flex align-items-center pb-2 h-100" style="justify-content: center;">
                                 <div class="d-flex flex-column h-100">
                                     <h3 class="card-title">
-                                    <img src="_static/quantum-computing.png" class="img-fluid" style="max-width: 66px;"></img>
+                                    <img src="_static/hubs/QC.svg" class="img-fluid" style="max-width: 66px;"></img>
```
Comment (josh146) [inline demonstrations.rst:122]:
```suggestion
                                    <img src="_static/hubs/QC.svg" class="img-fluid" style="max-width: 110px;"></img>
```

**Review by josh146 — APPROVED** [review]
Thanks @ikurecic! Just one minor suggestion to fix spacing consistency issues

**Comment by ikurecic** [comment]
Hey @josh146 , I just added (and changed) the images Tarik shared with me.

**Review by josh146 — APPROVED** [review]
Looks great 😍

**Pull request closed without merging**